rust/schema/annotations.py

```diff
@@ -1945,4 +1944,4 @@ class FormatArgument(Locatable):
 
 @annotate(Item, add_bases=(Addressable,))
 class _:
-    pass
+    expanded: optional[AstNode] | child | rust.detach | doc("expanded attribute or procedural macro call of this item")
```

Comment:
Would it be better to have a new base class, say `Expandable`, for this?

Reply:
considering that `MacroCall` is also an `Item`, and that the only remaining case to cover (derive macros) also applies to `Item`s, I don't see the need for another class in the hierarchy. What do you think would be the advantage of such an additional class?